src/unstract/sdk/adapters/ocr/google_document_ai/src/google_document_ai.py

```diff
@@ -95,10 +96,13 @@ def _get_request_headers(self) -> dict[str, Any]:
 
     """ Detect the mime type from the file content """
 
-    def _get_input_file_type_mime(self, input_file_path: str) -> str:
-        with open(input_file_path, mode="rb") as file_obj:
-            sample_contents = file_obj.read(100)
-            file_type = filetype.guess(sample_contents)
+    def _get_input_file_type_mime(
```

Comment:
NIT: @gaya3-zipstack please use the method from `/utils` instead which uses magic

Reply:
I am just refactoring existing function @chandrasekharan-zipstack I see more than one function for this and both of them being used. Will have to take the cleanup later.

Reply:
@gaya3-zipstack I suggested this for you to avoid the unnecessary work on this function - worst case, can you add a TODO comment to perform this cleanup later?

Reply:
Done.